index.html

```diff
@@ -0,0 +1,249 @@
+            voluptates neque ex id vel dolor soluta consectetur nam quidem
+            eligendi, nisi esse ad cum, repudiandae perspiciatis ullam tempore
+            sed officiis! Eum, quo, iusto odit reiciendis ducimus dolorum
+            dolores magnam earum eligendi quasi odio eaque possimus dolorem!
+            Architecto sequi qui voluptas accusamus illo magni debitis quasi
+            deleniti, possimus similique vitae hic quas non nesciunt,
+            consectetur expedita, aut iste veritatis iure molestiae.
+          </p>
+          <code
+            >function capitalizeFirstLetter(string) { return
+            string.charAt(0).toUpperCase() + string.slice(1); }
+            console.log(capitalizeFirstLetter('foo')); // Foo</code
+          >
+          <ul>
+            <li>js programming</li>
+          </ul>
+        </section>
+        <section class="main-section" id="section3">
+          <header>
+            <h1>section3</h1>
+          </header>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. In
+            temporibus quibusdam vitae assumenda! Ullam officiis repellendus
+            modi omnis iusto? Quam quaerat incidunt qui labore inventore
+            recusandae minus iste harum voluptatem.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestiae
+            saepe nesciunt nisi officia soluta unde placeat eos! Doloribus,
+            voluptatibus quos quia sunt ad, architecto saepe cupiditate nisi aut
+            tempore velit voluptate, molestiae ut? Modi rem, dolorem obcaecati
+            neque qui, asperiores nam ipsam, voluptates error hic excepturi
+            porro! Dolore, vitae doloremque? Aperiam repellat asperiores placeat
+            repudiandae, obcaecati minima maxime vel omnis amet! Consequuntur
+            libero dolorem pariatur distinctio non exercitationem numquam
+            voluptates neque ex id vel dolor soluta consectetur nam quidem
+            eligendi, nisi esse ad cum, repudiandae perspiciatis ullam tempore
+            sed officiis! Eum, quo, iusto odit reiciendis ducimus dolorum
+            dolores magnam earum eligendi quasi odio eaque possimus dolorem!
+            Architecto sequi qui voluptas accusamus illo magni debitis quasi
+            deleniti, possimus similique vitae hic quas non nesciunt,
+            consectetur expedita, aut iste veritatis iure molestiae.
+          </p>
+          <code
+            >function capitalizeFirstLetter(string) { return
+            string.charAt(0).toUpperCase() + string.slice(1); }
+            console.log(capitalizeFirstLetter('foo')); // Foo</code
+          >
+          <ul>
+            <li>js programming</li>
+          </ul>
+        </section>
+        <section class="main-section" id="section4">
+          <header>
+            <h1>section4</h1>
+          </header>
```

Comment:
I believe that the implementation of an h2 tag for design purposes would be more effective in this particular context. Using this approach would likely result in a more polished and professional presentation.

Reply:
- [ ] Good reason for change in a clear comment, great work 💯